packages/mui-base/src/useTab/useTab.test.tsx

```diff
@@ -18,7 +18,7 @@ describe('useTab', () => {
 
       function Test() {
         return (
-          <Tabs>
+          <Tabs defaultValue={1}>
```

Comment:
Was this related to this change?

Reply:
Yup ~ `Tabs` internally uses `useList`, so the controllable prop `selectedValues` was triggering the new warning